docs/components/base/fake.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Configure a Fake Base"
+linkTitle: "fake"
+weight: 50
+type: "docs"
+description: "Configure a fake base to use for testing without physical hardware."
+tags: ["base", "components"]
+# SMEs: Steve B
+---
+
+Configure a `fake` base to test implementing a base component on your robot without any physical hardware.
+
+Configure a `fake` base as follows:
+
+{{< tabs name="Configure a Fake Base" >}}
+{{% tab name="JSON Template" %}}
+
+```json-viam {class="line-numbers linkable-line-numbers"}
+{
+    "name": <"base_name">,
+    "type": "base",
+    "model": "fake",
+    "attributes": {}
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
```

Comment:
I'd say add a quick screenshot of config builder for consistency and so that there isn't a tab set of one single tab

Reply:
ok, done!